dotlottie-rs/src/fms/functions.rs

```diff
@@ -180,7 +152,7 @@ pub fn get_theme(bytes: &[u8], theme_id: &str) -> Result<String, DotLottieError>
 pub fn get_state_machine(bytes: &[u8], state_machine_id: &str) -> Result<String, DotLottieError> {
     let mut archive =
         ZipArchive::new(io::Cursor::new(bytes)).map_err(|_| DotLottieError::ArchiveOpenError)?;
-    let search_file_name = format!("states/{}.json", state_machine_id);
+    let search_file_name = format!("s/{}.json", state_machine_id);
```

Comment:
@theashraf get_animation supports both versions of the dotLottie format, shoudn't `get_state_machine` & `get_theme support` both versions too?